testing/src/main/java/io/grpc/internal/testing/AbstractTransportTest.java

```diff
@@ -452,6 +453,12 @@ public void ping_afterTermination() throws Exception {
     }
     verify(mockPingCallback, timeout(TIMEOUT_MS)).onFailure(throwableCaptor.capture());
     Status status = Status.fromThrowable(throwableCaptor.getValue());
+    status = Status.UNKNOWN.withCause(new ClosedChannelException());
```

Comment:
I ll remove that. just want to trigger a failure to see what the output looks like on CI.